sweep/walletsweep.go

```diff
@@ -179,27 +179,28 @@ type DeliveryAddr struct {
 }
 
 // CraftSweepAllTx attempts to craft a WalletSweepPackage which will allow the
-// caller to sweep ALL outputs within the wallet to a list of outputs. Any
-// leftover amount after these outputs and transaction fee, is sent to a single
-// output, as specified by the change address. The sweep transaction will be
-// crafted with the target fee rate, and will use the utxoSource and
-// outputLeaser as sources for wallet funds.
+// caller to sweep ALL funds in All or SELECT outputs within the wallet to a
+// list of outputs. Any leftover amount after these outputs and transaction fee,
+// is sent to a single output, as specified by the change address. The sweep
+// transaction will be crafted with the target fee rate, and will use the
+// utxoSource and outputLeaser as sources for wallet funds.
 func CraftSweepAllTx(feeRate, maxFeeRate chainfee.SatPerKWeight,
 	blockHeight uint32, deliveryAddrs []DeliveryAddr,
 	changeAddr btcutil.Address, coinSelectLocker CoinSelectionLocker,
 	utxoSource UtxoSource, outputLeaser OutputLeaser,
-	signer input.Signer, minConfs int32) (*WalletSweepPackage, error) {
+	signer input.Signer, minConfs int32,
+	selectUtxos []wire.OutPoint) (*WalletSweepPackage, error) {
```

Comment:
Should we again make this a variadic argument? Then we wouldn't need to add `, nil, ` to every call site of this function.

Reply:
Although this currently has only one call site (the sendcoins rpc function). But maybe there are external packages using it.